packages/kbn-es/src/utils/docker.ts

```diff
@@ -714,7 +714,7 @@ export async function runDockerContainer(log: ToolingLog, options: DockerOptions
   const dockerCmd = resolveDockerCmd(options, image);
 
   log.info(chalk.dim(`docker ${dockerCmd.join(' ')}`));
-  return await execa('docker', dockerCmd, {
+  await execa('docker', dockerCmd, {
```

Comment:
@delanni I believe it is safe to remove `return` since it will be resolved on process exit  and I don't how it changes the logic of `cluster.runDocker(..)`

Did I miss something?

Reply:
I think it's safe to change too, we probably don't want much to do with the PID of the docker run, we're stopping the cluster by names anyway.